model/coverage.py

```diff
@@ -235,24 +239,32 @@ def update(self, start=None, finish=None, achievements=None,
         """
 
         if start is not None:
-            if start is self.NO_VALUE:
+            if start is self.CLEAR_VALUE:
                 # In most cases, None is not a valid value for
                 # Timestamp.start, but this can be overridden.
                 start = None
             self.start = start
         if finish is not None:
-            if finish is self.NO_VALUE:
+            if finish is self.CLEAR_VALUE:
                 # In most cases, None is not a valid value for
                 # Timestamp.finish, but this can be overridden.
                 finish = None
             self.finish = finish
         if achievements is not None:
+            if achievements is self.CLEAR_VALUE:
+                achievements = None
             self.achievements = achievements
         if counter is not None:
+            if counter is self.CLEAR_VALUE:
+                counter = None
             self.counter = counter
 
-        # Unlike the other fields, None is a realistic value for
-        # .exception
+        # Unlike the other fields, None is the default value for
+        # .exception, so passing in None to mean "use the default" and
+        # None to mean "no exception" mean the same thing. But we'll
+        # support CLEAR_VALUE anyway.
+        if exception is self.CLEAR_VALUE:
+            exception = None
```

Comment:
This doesn't seem like a super useful feature but I figured I'd add it to be consistent.